plugins/events-node/api-report.md

```diff
@@ -18,32 +32,56 @@ export interface EventParams<TPayload = unknown> {
   topic: string;
 }
 
-// @public
+// @public @deprecated
 export interface EventPublisher {
-  // (undocumented)
+  // @deprecated (undocumented)
   setEventBroker(eventBroker: EventBroker): Promise<void>;
 }
 
 // @public
-export abstract class EventRouter implements EventPublisher, EventSubscriber {
+export abstract class EventRouter {
+  protected constructor(options: {
+    eventService: EventService;
+    topics: string[];
+  });
   // (undocumented)
   protected abstract determineDestinationTopic(
     params: EventParams,
   ): string | undefined;
   // (undocumented)
   onEvent(params: EventParams): Promise<void>;
-  // (undocumented)
-  setEventBroker(eventBroker: EventBroker): Promise<void>;
-  // (undocumented)
-  abstract supportsEventTopics(): string[];
+  subscribe(): Promise<void>;
 }
 
 // @public
+export interface EventService {
+  publish(params: EventParams): Promise<void>;
+  subscribe(options: EventSubscriptionOptions): Promise<void>;
+}
+
+// @public
+const eventServiceFactory: () => ServiceFactory<EventService, 'root'>;
+export default eventServiceFactory;
+export { eventServiceFactory };
+
+// @public
+export const eventServiceRef: ServiceRef<EventService, 'root'>;
```

Comment:
```suggestion
export const eventsServiceRef: ServiceRef<EventsService, 'root'>;
```

This is what freben meant by https://github.com/backstage/backstage/pull/22344#discussion_r1471166305. Basically renaming from "event" to "events" everywhere. It works quite a bit better for the local variable name